packages/common/utilities/info.ts

```diff
@@ -0,0 +1,25 @@
+import { isProduction } from '../constants/environments';
+
+const info = (condition: unknown, message: string): void => {
+  // don't do anything in production
+  // wrapping in production check for better dead code elimination
+  if (!isProduction()) {
+    // condition passed: do not log
+    if (condition) {
+      return;
+    }
+
+    // Condition not passed
+    const text = `Info: ${message}`;
+
+    // check console for IE9 support which provides console
+    // only with open devtools
+    if (typeof console !== 'undefined') {
+      // we want to log a warning; so usage of Console is required
+      // eslint-disable-next-line no-console
```

Comment:
I believe it's better to stick the reason to the disabling comment:

```suggestion
      // eslint-disable-next-line no-console -- We want to log a warning; so usage of Console is required.
```

Copy-pasting will be safer and we can do fulltext search on all disabling comments that do not have a reason next to it.

https://eslint.org/docs/latest/use/configure/rules#comment-descriptions

Reply:
Fixed.